backend/frontend/src/pages/landing/Register.js

```diff
@@ -46,33 +52,124 @@ function Register() {
+        <div style={{width: 960, height: 1005, left: 550, top: 0, position: 'absolute', background: 'white', borderTopLeftRadius: 40, borderTopRightRadius: 40, overflow: 'hidden'}}>
+            <div style={{width: 339, height: 53, left: 304, top: 140, position: 'absolute', color: '#2C2A2A', fontSize: 40, fontFamily: 'Inter', fontWeight: '700', wordWrap: 'break-word'}}>Create Account</div>
+                 {/* <button type="submit" className="btn btn-primary">Register</button> */}
+                <Button
+                  variant="contained"
+                  color="primary"
+                  onClick={handleSubmit}
+                  className="register-button"
+                  style={{
+                    width: 473,
+                    height: 53,
+                    paddingLeft: 40,
+                    paddingRight: 40,
+                    paddingTop: 15,
+                    paddingBottom: 15,
+                    left: 312,
+                    top: 570,
+                    position: 'absolute',
+                    borderRadius: 8,
+                    justifyContent: 'center',
+                    alignItems: 'center',
+                    gap: 10,
+                    display: 'inline-flex',
+                    cursor: 'pointer', // Add this to make it look clickable
+                    fontSize: 16,
+                    fontFamily: 'Inter',
+                    fontWeight: '700',
+                    wordWrap: 'break-word',
+                    textTransform: 'none',
+                    backgroundColor: 'purple',
+                  }}
+                >
+                  Create Account
+              </Button>
+            <div style={{ width: 473, height: 59, left: 304, top: 240, position: 'absolute', background: 'white', borderRadius: 8, padding: 8 }}>
+                <TextField
+                    id="register-username"
+                    label="Username"
+                    onChange={(e) => setUsername(e.target.value)}
+                    style={{
+                        width: '100%',
+                        height: '100%', // Set height to '100%'
+                        color: '#7C7A7A',
+                        borderRadius: 8,
+                        fontSize: 16,
+                        fontFamily: 'Inter',
+                        fontWeight: '500',
+                        wordWrap: 'break-word',
+                        boxSizing: 'border-box', // Include padding and borders in the total width and height
+                    }}
+                />
+            </div>
+              <div style={{width: 473, height: 59, left: 304, top: 320, position: 'absolute', background: 'white', borderRadius: 8, padding: 8}}>
+                    <TextField
+                            id="register-email"
+                            label="E-Mail"
+                            onChange={(e) => setEmail(e.target.value)}
+                            style={{
```

Comment:
As far as i can see, register-username and register-email have same style codes. Is it possible to use css and className here in these input components?

Reply:
It's done!